README.md

```diff
@@ -23,16 +23,34 @@ LanguageTool-Rust (LTRS) is both an executable and a Rust library that aims to p
 
 *Disclaimer: the current work relies on an approximation of the LanguageTool API. We try to avoid breaking changes as much as possible, but we still highly depend on the future evolutions of LanguageTool.*
 
-## CLI Reference
-
-![Screenshot from CLI](https://raw.githubusercontent.com/jeertmans/languagetool-rust/main/img/screenshot.svg)
+## Installation
 
-The command line interface of LTRS allows to very quickly use any LanguageTool server to check for grammar and style errors. You can install the latest version with `cargo`:
+You can install the latest version with `cargo`.
 
 ```bash
 > cargo install languagetool-rust --features full
 ```
 
+And this to your `.bashrc` or `.zshrc`.
+
+```bash
+export PATH=$PATH:$HOME/.cargo/bin
+```
```

Comment:
```suggestion
```

I don't think LTRs needs to tell that. This should be performed only once, and most likely when installing Cargo.